util/liblog/liblog_test.go

```diff
@@ -14,10 +14,6 @@ type mock struct {
 	args        []interface{}
 }
 
-func (m *mock) Impl() interface{} {
-	return m
-}
-
```

Comment:
This only got added recently. Now it's removed again. 

What's the decision here? Use `cosmossdk.io/logger` or the CometBFT one?

Reply:
I think as per  docs we need to use the cosmossdk.io/log